game/pokemon.py

```diff
@@ -6,6 +6,8 @@
 
 
 class Pokemon:
+    """Holds information about Pokemon in current instance of the application"""
```

Comment:
Minor: since the word 'Pokemon` can refer to one or several Pokemon, consider changing this to refer to 'a Pokemon`